core/indexer/indexer.go

```diff
@@ -110,6 +114,33 @@ func (i *indexer) Close(ctx context.Context) (err error) {
 	return nil
 }
 
+func (i *indexer) RemoveAclIndexes(spaceId string) (err error) {
+	ids, _, err := i.store.QueryObjectIDs(database.Query{
+		Filters: []*model.BlockContentDataviewFilter{
+			{
+				RelationKey: bundle.RelationKeySpaceId.String(),
+				Condition:   model.BlockContentDataviewFilter_Equal,
+				Value:       pbtypes.String(spaceId),
+			},
+			{
+				RelationKey: bundle.RelationKeyType.String(),
+				Condition:   model.BlockContentDataviewFilter_Equal,
+				Value:       pbtypes.String(bundle.TypeKeyParticipant.String()),
```

Comment:
we store type id here, not type key. easier to compare with layout participant here

Reply:
fixed